### PR TITLE
ADD **kwargs in fmin facade passed to SMAC facade

### PR DESCRIPTION
**Pull request by mlindauer**
see title

### CODE REVIEW

File: smac/facade/func_facade.py [hunk 1]
```diff
@@ -36,6 +37,8 @@ def fmin_smac(func: callable,
         Maximum number of function evaluations.
     rng : np.random.RandomState, optional
             Random number generator used by SMAC.
+    **kwargs:
+        arguments passed to SMAC facade
```
Comment (mfeurer) [inline smac/facade/func_facade.py]: I suggest to not use the word facade here, it's quite confusing when you just want to minimize a function. How about `Arguments passed to the optimizer class.`, ideally with a link.

**Comment by mlindauer** [comment]
@mfeurer please review again

File: smac/facade/func_facade.py [hunk 2]
```diff
@@ -15,31 +16,36 @@
...
+    bounds : typing.List[typing.List[float]]
         ``(min, max)`` pairs for each element in ``x``, defining the bound on
         that parameters.
     maxfun : int, optional
         Maximum number of function evaluations.
     rng : np.random.RandomState, optional
             Random number generator used by SMAC.
+    scenario_args: typing.Mapping[str,typing.Any]
+        Arguments passed to the scenario
+        See smac.scenario.scenario.Scenario
```
Comment (mfeurer) [inline smac/facade/func_facade.py]: I think you need to apply some special syntax to link to the class, for example like here: https://github.com/automl/SMAC3/blob/master/smac/facade/smac_facade.py#L78

Comment (mlindauer) [reply smac/facade/func_facade.py]: hmm.. I would like to avoid to link to github. I think docstrings should be independent of github.

Comment (mlindauer) [reply smac/facade/func_facade.py]: Done

**Pull request merged**